README.md

```diff
@@ -49,14 +49,20 @@ Make sure to maintain your local fork going forward so it stays up-to-date with
 
 We're happy to help answer any questions you may have and give you feedback on your articles in the [contributors chat room](https://gitter.im/freecodecamp/contributors).
 
-Finally, if you want to run a version of the Guide repository locally, follow these commands:
+Finally, if you want to run a version of the Guide repository locally, follow these steps:
 
+1. Ensure you have the `yarn` package manager installed `npm install -g yarn`
+2. Fork this repository
+3. :point_down: 
 ```sh
-git clone https://github.com/freeCodeCamp/guides.git
+git clone https://github.com/<YOUR GITHUB USERNAME>/guides.git
```

Comment:
@Bouncey - looks good, thanks for updating this! Only one really minor comment for consistency - we say "<your-username>" up above in the part about cloning the repo. We should change either that one to match how you have it here, or this one, but either way we should be consistent between the two. 😄